src/components/Capture/Document.tsx

```diff
@@ -150,7 +150,6 @@ class Document extends Component<Props> {
       className={theme.warningFallbackButton}
       onChange={this.handleFileSelected}
       accept="image/*"
-      capture
```

Comment:
This change in general will allow users on mobile devices to choose between native camera capture or uploading a file from their photo gallery. 
Is this an intentional UX change? We've so far wanted to avoid users being able to upload from their photo gallery if possible.

Reply:
Yes you're right. I've updated to explicitly declare a valid value (`user` or `environment`) for `capture` prop. But please note that for devices without a working media channel, the user can still be prompted with the photo gallery